src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelInternal.java

```diff
@@ -256,6 +262,14 @@ public CompletableFuture<Void> close() {
     return this.close(false);
   }
 
+  /**
+   * Attempts to close the channel by draining any outstanding data and rejecting further writes to
+   * the channel.
+   *
+   * @param drop if true, the channel will be dropped after all data is successfully committed.
+   * @throws SFException when the channel is invalid or closed, meaning that no new rows can be
```

Comment:
nit: lets keep the references to SFException consistent, earlier used `net.snowflake.ingest.utils.SFException`

Reply:
Ack